allennlp/data/vocabulary.py

```diff
@@ -417,6 +430,26 @@ def empty(cls) -> "Vocabulary":
         """
         return cls()
 
+    @staticmethod
+    def add_transformer_vocab(tokenizer: PreTrainedTokenizer) -> Dict[str, Dict]:
```

Comment:
I meant that this should be an instance method:

```suggestion
    def add_transformer_vocab(self, tokenizer: PreTrainedTokenizer, namespace: str = "tokens") -> None:
```

Reply:
done!